pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -47,43 +47,43 @@ func getPluginArgConversionScheme() *runtime.Scheme {
 }
 
 func Convert_v1beta1_Plugins_To_config_Plugins(in *v1beta1.Plugins, out *config.Plugins, s conversion.Scope) error {
-	if err := convertToInternalPluginSet(in.QueueSort, &out.QueueSort, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.QueueSort, &out.QueueSort, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.PreFilter, &out.PreFilter, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.PreFilter, &out.PreFilter, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.Filter, &out.Filter, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.Filter, &out.Filter, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.PostFilter, &out.PostFilter, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.PostFilter, &out.PostFilter, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.PreScore, &out.PreScore, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.PreScore, &out.PreScore, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.Score, &out.Score, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.Score, &out.Score, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.Reserve, &out.Reserve, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.Reserve, &out.Reserve, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.Permit, &out.Permit, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.Permit, &out.Permit, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.PreBind, &out.PreBind, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.PreBind, &out.PreBind, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.Bind, &out.Bind, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.Bind, &out.Bind, s); err != nil {
 		return err
 	}
-	if err := convertToInternalPluginSet(in.PostBind, &out.PostBind, s); err != nil {
+	if err := Convert_v1beta1_PluginSet_To_config_PluginSet(in.PostBind, &out.PostBind, s); err != nil {
 		return err
 	}
 	return nil
 }
 
-func convertToInternalPluginSet(in *v1beta1.PluginSet, out *config.PluginSet, s conversion.Scope) error {
```

Comment:
I see that I wasn't clear. I meant that you should prefer the autogenerated functions

Reply:
done.